src/sortStudents.ts

```diff
@@ -1,16 +1,69 @@
-
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+function averageGrade(grades: number[]) : number {
```

Comment:
```suggestion
function getAverageGrade(grades: number[]) : number {
```
Get used to name functions according to these rules:
![image](https://user-images.githubusercontent.com/96409563/213545172-ba2577fb-8891-4a2e-b2f7-acf9a48ce617.png)